src/cvode/cvode.c

```diff
@@ -4035,17 +4047,18 @@ static int cvRcheck2(CVodeMem cv_mem)
 
 static int cvRcheck3(CVodeMem cv_mem)
 {
+  SUNAssignSUNCTX(CV_SUNCTX);
   int i, ier, retval;
 
   /* Set thi = tn or tout, whichever comes first; set y = y(thi). */
   if (cv_mem->cv_taskc == CV_ONE_STEP) {
     cv_mem->cv_thi = cv_mem->cv_tn;
-    N_VScale(ONE, cv_mem->cv_zn[0], cv_mem->cv_y);
+    SUNCheckCallLastErrNoRet(N_VScale(ONE, cv_mem->cv_zn[0], cv_mem->cv_y));
   }
   if (cv_mem->cv_taskc == CV_NORMAL) {
     if ( (cv_mem->cv_toutc - cv_mem->cv_tn)*cv_mem->cv_h >= ZERO) {
       cv_mem->cv_thi = cv_mem->cv_tn;
-      N_VScale(ONE, cv_mem->cv_zn[0], cv_mem->cv_y);
+      SUNCheckCallLastErrNoRet(N_VScale(ONE, cv_mem->cv_zn[0], cv_mem->cv_y));
     } else {
       cv_mem->cv_thi = cv_mem->cv_toutc;
       (void) CVodeGetDky(cv_mem, cv_mem->cv_thi, 0, cv_mem->cv_y);
```

Comment:
```suggestion
      SUNCheckCallNoRet(CVodeGetDky(cv_mem, cv_mem->cv_thi, 0, cv_mem->cv_y));
```

Reply:
Actually, it looks like multiple other functions inside this routine are missing error checking as well (`cv_gfun`, `cvRootfind`, another `CVodeGetDky`).

Reply:
`CVodeGetDky` does not return a `SUNErrCode`, that is why its not checked.